### ISSUE TITLE
problem with Inventorio offhand

### ISSUE THREAD

**Issue by G4lds** [cc-tweaked/cc-restitched#141]
### Minecraft Version

1.19.x

### Version

1.101.2 and 1.102.0

### Details

when i am using a pocket computer on one of the offhand slots of the Inventorio mod the computer shutdown on a random short period of time

I have confirmed this bug in 1.19.2 and 1.19.3:

mod list:
CC: Tweaked
Fabric API
Fabric Language Kotlin
Inventorio

**Comment by Merith-TK** [comment]
Looking at the mod, probably is because the computer is not in the inventory?? Pocket Computers have weird inventory logic to determine when they should be turned on, and generate an ComputerID (legit, crafting one gives it an CID rather that it getting assigned one on first interaction like every other computer block)

it might just need to be in your hotbar to keep functional,

**Comment by G4lds** [comment]
yeah, i also think it is not recognizing inventorio slots, it works fine on the hotbar.
what i wanted to know is if i can fix this behavior in game through programming or i ill need to do some tweaking on the mod files

**Comment by Merith-TK** [comment]
It would require modifying the source code for either CC:R or inventorio
